bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -3288,3 +3288,30 @@ pimcore.helpers.treeDragDropValidate = function (node, oldParent, newParent) {
 
     return true;
 };
+
+pimcore.helpers.sendRequestWithoutDefaultHeaders = function (
```

Comment:
```suggestion
// Sends an Ajax request, it is recommended to be used when doing simple calls or to third-party services, in contrast to Ext.Ajax.request which by default sends extra info (eg. custom header) that is usually needed to be working within Pimcore interface.
pimcore.helpers.sendRequest = function (
```
suggesting to rename the function to a shorter name and add a comment or so, wdyt?